packages/ERTP/test/unitTests/test-issuerObj.js

```diff
@@ -105,6 +107,46 @@ test('issuer.makeEmptyPurse', t => {
   }
 });
 
+test('issuer.deposit', t => {
+  t.plan(2);
+  const { issuer, mint, amountMath } = produceIssuer('fungible');
+  const fungible25 = amountMath.make(25);
+
+  const purse = issuer.makeEmptyPurse('my new purse');
```

Comment:
Purses have no names now
```suggestion
  const purse = issuer.makeEmptyPurse();
```

Reply:
done